src/com/rakovets/course/practice/string/Task01.java

```diff
@@ -44,7 +46,9 @@ static String getSurname(String fullName) {
 		//TODO
 		// Код, решающий задачу пишем ниже, при этом используя параметры метода
 		// Для проверки решения необходимо запустить @Test для данного class (в директории test)
-		return null;
+		int positionSpace = fullName.indexOf(" ");
+		String surname = fullName.substring(positionSpace+1);

```

Comment:
Violation of Code Convention!
```suggestion
		String surname = fullName.substring(positionSpace + 1);
```